src/EndpointV2/EndpointV2SerializerTrait.php

```diff
@@ -174,6 +174,7 @@ private function applyAuthSchemeToCommand($endpoint, $command)
     private function selectAuthScheme($authSchemes)
     {
         $validAuthSchemes = ['sigv4', 'sigv4a' ];
+        $unsupportedScheme = '';
```

Comment:
We should get rid of this variable entirely, and have the error message instead use the variables $authSchemes and $validAuthSchemes to show what is supported by the endpoint and what we support.  We will need a way of remembering to add to the above array as we add auth schemes as well, or as token providers become available in endpoints 2.0.